frontend/src/components/EasyModal.tsx

```diff
@@ -32,17 +33,42 @@ function EasyModal(props: EasyModalProps) {
 
   return (
     <BaseModal modalRef={modalRef}>
-      <div className="w-[332px] flex flex-col gap-8">
-        <div className="flex flex-col gap-6">
-          <div className="flex flex-row justify-between items-center">
-            <h2>{title}</h2>
-            {showCloseButton && <RightCloseButton onClick={handleClose} />}
-          </div>
-          {children}
-        </div>
-        <PrimaryButton onClick={handleSave} fullWidth>
-          <span className="text-white normal-semibold">Legg til</span>
-        </PrimaryButton>
+      <div className="w-[299px] md:inset-0 h-[calc(100%-1rem)] max-h-full relative max-w-2xl">
```

Comment:
Her har også noe gammel kode sneket seg inn 😅 For i stedet brukte vi: 

```suggestion
      <div className="w-[332px]">
```

Reply:
Darn 😓 Fra nå av ikke merge branch-ut-av-egen-branch-uten-remote-sync 🔀